projects/onto-search/src/lib/onto-search-facet/range-facet/range-facet.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="range-facet" appCypressData="onto-range-facet">
+<onto-range-histogram *ngIf="data.showHistogram" [data]="data" [containerWidth]="containerWidth"></onto-range-histogram>
```

Comment:
Why do you need to pass this `containerWidth` from outside? Can't you find it programatically?

Reply:
done